peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -137,6 +137,8 @@ <h5>{% translate 'Related documents' %}</h5>
             </div>
           {% endblock %}
 
+          {% include 'peachjam/_notices.html' with messages=notices %}
```

Comment:
@nickmwangemi pls put the notices beneath the document title, and above the tabs.